app/src/main/java/com/asfoundation/wallet/iab/presentation/icon/IcCheck.kt

```diff
@@ -0,0 +1,85 @@
+package com.asfoundation.wallet.iab.presentation.icon
+
+import androidx.compose.foundation.Image
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.PathFillType.Companion.NonZero
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap.Companion.Butt
+import androidx.compose.ui.graphics.StrokeJoin.Companion.Miter
+import androidx.compose.ui.graphics.vector.ImageVector.Builder
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.appcoins.wallet.ui.common.theme.WalletTheme
+
+@Composable
+@Preview
+private fun PreviewGetIcCheck(modifier: Modifier = Modifier) {
+  WalletTheme {
+    Image(
+      imageVector = getIcCheck(),
+      contentDescription = null
+    )
+  }
+}
+
+@Composable
+fun getIcCheck() =
```

Comment:
Move the compose vector images to the ui.common module to be accessible by all modules